Makefile

```diff
@@ -446,6 +446,22 @@ coverage-report: $(SUMMARY_COVER_PROFILE)
 	@printf $(COLOR) "Generate HTML report from $(SUMMARY_COVER_PROFILE) to $(SUMMARY_COVER_PROFILE).html..."
 	@go tool cover -html=$(SUMMARY_COVER_PROFILE) -o $(SUMMARY_COVER_PROFILE).html
 
+upload-test-results:
+	$(foreach FILE,$(wildcard $(TEST_OUTPUT_ROOT)/*.junit.xml),\
```

Comment:
I think you want a for loop in shell, not a make foreach construct. actually I'd want to move this all into a script in the repo and just call it here, that'll be easier to maintain

Reply:
> I think you want a for loop in shell, not a make foreach construct.

Why's that? There are two other make targets here already that I was mimicking.

> actually I'd want to move this all into a script in the repo and just call it here, that'll be easier to maintain

Oh? What would make it easier to maintain? I don't believe we have any dev scripts anymore. We actually got rid of one in the past (see https://github.com/temporalio/temporal/pull/4938/files#diff-d22690e5ffc7fbcc6c0d71f7ec6914992106d70a0f0d83a09e7f665056059f53L315).

Reply:
That's a fair point. I think there's a simplicity to not invoking separate scripts that I really appreciate; but - to your point - it comes at a cost. Alright, I'll move it to a separate shell script.

Reply:
@dnr It's a separate shell script now.